### PR TITLE
pifi: init at 0.4.11

### PR DESCRIPTION
**Pull request by kmein**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/pifi/default.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+bundlerEnv rec {
+  name = "pifi-${version}";
+
+  version = (import ./gemset.nix).pifi.version;
+  inherit ruby;
+  # expects Gemfile, Gemfile.lock and gemset.nix in the same directory
+  gemdir = ./.;
+
+  meta = with lib; {
+    description = "MPD web client to listen to radio, written in React and Sinatra.";
```
Comment (fabaff) [inline pkgs/applications/audio/pifi/default.nix:12]:
```suggestion
    description = "MPD web client to listen to radio, written in React and Sinatra";
```

see https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes

File: pkgs/applications/audio/pifi/default.nix [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    homepage = "https://github.com/rccavalcanti/pifi-radio";
+    license = with licenses; gpl3;
```
Comment (fabaff) [inline pkgs/applications/audio/pifi/default.nix:14]: The license is [GPLv3](https://github.com/rccavalcanti/pifi-radio/blob/master/pifi.gemspec#L7) only.

```suggestion
    license = with licenses; gpl3Only;
```

**Review by fabaff — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/pifi/default.nix [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv rec {
+  name = "pifi-${version}";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/pifi/default.nix:4]:
```suggestion
  pname = "pifi";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/pifi/gemset.nix [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    groups = ["default"];
+    platforms = [];
+    source = {
+      remotes = ["https://rubygems.org"];
+      sha256 = "0rn8z8hda4h41a64l0zhkiwz2vxw9b1nb70gl37h1dg2k874yrlv";
+      type = "gem";
+    };
+    version = "2.0.10";
+  };
+}
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/pifi/gemset.nix:137]: Missing final new line.

**Comment by SuperSandro2000** [comment]
Note sure why this happens:

```
$ nix-env -f /home/hotpi/.cache/nixpkgs-review/pr-110814-1/nixpkgs -qaP --xml --out-path --show-trace                                                                                       error: stack overflow (possible infinite recursion)                                                                                                                                         $ git worktree prune                                                                                                                                                                        Traceback (most recent call last):                                                                                                                                                            File "/nix/store/kxyf9jaz17dxmx23psp6kiqxi56b2xlp-nixpkgs-review-2.5.0/bin/.nixpkgs-review-wrapped", line 9, in <module>                                                                      sys.exit(main())                                                                                                                                                                          File "/nix/store/kxyf9jaz17dxmx23psp6kiqxi56b2xlp-nixpkgs-review-2.5.0/lib/python3.8/site-packages/nixpkgs_review/__init__.py", line 10, in main                                              cli.main(command, args)                                                                                                                                                                   File "/nix/store/kxyf9jaz17dxmx23psp6kiqxi56b2xlp-nixpkgs-review-2.5.0/lib/python3.8/site-packages/nixpkgs_review/cli/__init__.py", line 242, in main                                         args.func(args)                                                                                                                                                                           File "/nix/store/kxyf9jaz17dxmx23psp6kiqxi56b2xlp-nixpkgs-review-2.5.0/lib/python3.8/site-packages/nixpkgs_review/cli/pr.py", line 62, in pr_command                                          contexts.append((pr, builddir.path, review.build_pr(pr)))                                                                                                                                 File "/nix/store/kxyf9jaz17dxmx23psp6kiqxi56b2xlp-nixpkgs-review-2.5.0/lib/python3.8/site-packages/nixpkgs_review/review.py", line 212, in build_pr                                           return self.build_commit(base_rev, pr_rev)                                                                                                                                                File "/nix/store/kxyf9jaz17dxmx23psp6kiqxi56b2xlp-nixpkgs-review-2.5.0/lib/python3.8/site-packages/nixpkgs_review/review.py", line 153, in build_commit                                       base_packages = list_packages(str(self.worktree_dir()))                                                                                                                                   File "/nix/store/kxyf9jaz17dxmx23psp6kiqxi56b2xlp-nixpkgs-review-2.5.0/lib/python3.8/site-packages/nixpkgs_review/review.py", line 316, in list_packages                                      return parse_packages_xml(nix_env.stdout)                                                                                                                                                 File "/nix/store/kxyf9jaz17dxmx23psp6kiqxi56b2xlp-nixpkgs-review-2.5.0/lib/python3.8/site-packages/nixpkgs_review/review.py", line 257, in parse_packages_xml                                 for (event, elem) in context:                                                                                                                                                             File "/nix/store/2wwb85hrvvfzkhrlwcj3pqi60nlbi9cq-python3-3.8.7/lib/python3.8/xml/etree/ElementTree.py", line 1233, in iterator                                                               root = pullparser._close_and_return_root()                                                                                                                                                File "/nix/store/2wwb85hrvvfzkhrlwcj3pqi60nlbi9cq-python3-3.8.7/lib/python3.8/xml/etree/ElementTree.py", line 1280, in _close_and_return_root                                                 root = self._parser.close()
xml.etree.ElementTree.ParseError: no element found: line 23201, column 0
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110814` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>pifi</li>
</ul>
</details>

**Pull request merged**